test-packages/test-utils/src/utils/download-ui5-resources.ts

```diff
@@ -17,29 +17,32 @@ async function getLibs(version: TestModelVersion): Promise<string[]> {
   // For now we assume the libraries are the same in all versions, when we support newer versions we should
   // do a better check here
   let versionInMetadataURL: string = version;
-  if (versionInMetadataURL !== "1.76.0") {
-    versionInMetadataURL = "1.76.0";
+  if (versionInMetadataURL !== "1.71.49") {
+    versionInMetadataURL = "1.71.49";
   }
-  const url = `https://unpkg.com/@sapui5/distribution-metadata@${versionInMetadataURL}/metadata.json`;
+  //const url = `https://unpkg.com/@sapui5/distribution-metadata@${versionInMetadataURL}/metadata.json`;
```

Comment:
remove commented out "old" line?

Reply:
ACK - but I like so much to keep those links to remember me... 😄 - will remove it!